src/transformers/pipelines/__init__.py

```diff
@@ -480,6 +482,11 @@ def pipeline(
         use_auth_token (`str` or *bool*, *optional*):
             The token to use as HTTP bearer authorization for remote files. If `True`, will use the token generated
             when running `transformers-cli login` (stored in `~/.huggingface`).
+        device_map:
+            Sent directly as `model_kwargs` (just a simpler shortcut) to load the model automatically on multiple GPUs if `accelerate`
+            is available (do not use `device_map` AND `device` at the same time).
```

Comment:
It also does offload on CPU disk, worth mentioning? I'd also add
```
To have Accelerate compute the most optimized `device_map` automatically, set `device_map="auto"`.
```

Reply:
Ideally I would forward to a full documentation for what this does, any nice way to link to this: https://github.com/huggingface/transformers/blob/main/src/transformers/modeling_utils.py#L1679 Or maybe accelerate docs directly ?

I think this docstring could go out of date pretty fast so the idea here would be to have a bare minimum explanation, and reference a real fully documented place with a link if possible.

Reply:
We can't link to that specific arg in the doc yet (just raised an issue) ;-)
The general page in Accelerate for big models is https://huggingface.co/docs/accelerate/main/en/big_modeling otherwise.

Reply:
I'll link that for now, I think it's rather complete.